# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLContext.h
# Commit: adc841b3e204
# Full Hash: adc841b3e204564ee8515a87206eb317275844c4
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-02-21 09:51:10
# Regressor Bug: 1616715
# File Overlap Count: 1
# Description:
#   Bug 1616715 - SurfaceFromElement gets alpha Premult unless opt-in to NonPremult. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63421
# ==============================================================================

diff -r 7917b3a4a9a7 -r adc841b3e204 dom/canvas/WebGLContext.h
--- a/dom/canvas/WebGLContext.h	Fri Feb 21 03:11:23 2020 +0000
+++ b/dom/canvas/WebGLContext.h	Fri Feb 21 02:27:08 2020 +0000
@@ -374,9 +374,6 @@
 
   void SetCompositableHost(RefPtr<layers::CompositableHost>& aCompositableHost);
 
-  RefPtr<mozilla::gfx::SourceSurface> GetSurfaceSnapshot(
-      gfxAlphaType* out_alphaType);
-
   /**
    * An abstract base class to be implemented by callers wanting to be notified
    * that a refresh has occurred. Callers must ensure an observer is removed